### PR TITLE
Added alerts and buttons

### PR DESCRIPTION
**Pull request by haarsh24**
1. Added alerts and buttons.
2. Changes the folder structure for repo.

### CODE REVIEW

File: components/buttons.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/* Buttons */
+
+button{
+    font-size: 1.2rem;
+}
+.btns {
```
Comment (irajbhinde) [inline components/buttons.css:6]: could use a better naming here, like content-button

**Review by irajbhinde — COMMENTED** [review]
Looks good, if you can add descriptions for components, that'd be great.

File: components/buttons.css (hunk 1 above)
Comment (haarsh24) [reply components/buttons.css:6]: Sure

**Comment by haarsh24** [comment]
> Looks good, if you can add descriptions for components, that'd be great.

Yes , will add that !

File: components/alerts.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.alerts-div {
+}
```
Comment (xii-vi) [inline components/alerts.css:2]: why an empty class??

**Review by xii-vi — APPROVED** [review]
After improving the suggested changes, you can merge.

**Pull request merged**